### ISSUE TITLE
The latest 1.18.* build is not compatible with 1.18.1

### ISSUE THREAD

**Issue by dranorter** [Andrew6rant/InventoryTabs#42]
Version 0.8.1 crashes Minecraft 1.18.1 before the loading screen is even reached. The mod fails to find lwjgl.dll, and jemalloc.dll. Tested initially with several other mods, but also verified with just InventoryTabs, FabricAPI, and ClothConfig.

The old release "0.4.2" works fine on 1.18.1. I have not tested releases between the two.

As a temporary measure, releases currently marked "1.18.*" could be renamed to indicate they're only compatible with "1.18.2". Of course, 0.4.2 is missing compatibility with modded chests, and other fixes, so I'd prefer a real fix.

Log:
https://paste.ee/p/vPyHs

**Comment by Andrew6rant** [comment]
I'll look into getting this fixed.

Why are you using 1.18.1 anyway? Every single mod on that log has an 1.18.2 version (except No Null Processors, which is not needed on 1.18.2 since vanilla fixed the bug)

**Comment by dranorter** [comment]
My players complain when I update things, some of the people on the server are kids who have to have a parent help with updating. We made a collective decision to skip 1.18.2.

**Comment by Christian22023** [comment]
Works perfect on inventorytabs-0.5.0-1.18.x for Minecraft 1.18.1, but newer versions crash

@Andrew6rant

**Comment by Andrew6rant** [comment]
The code I wrote for dynamically detecting modded inventories at runtime (added in version 0.6.0) and the config configuration system (added in 0.7.0, overhauled in 0.8.0) is dependent on 1.18.2's `LifecycledResourceManager`. Minecraft's tag system as a whole changed a lot from 1.18.1-1.18.2, I would unfortunately have to rewrite large swaths of code to support 1.18.1.

I've marked all non-1.18.1 compatible versions on CurseForge properly.
